main.go

```diff
@@ -45,6 +45,7 @@ func newDefragCommand() *cobra.Command {
 
 	defragCmd.Flags().BoolVar(&globalCfg.compaction, "compaction", true, "whether execute compaction before the defragmentation (defaults to true)")
 
+	defragCmd.Flags().BoolVar(&globalCfg.remote, "remote", true, "whether execute on remote machine")
```

Comment:
Please also put this right under the option "`--cluster`' (line 25).  It's only used when `--cluster=true`.
```suggestion
	defragCmd.Flags().BoolVar(&globalCfg.excludeLocalhost, "exclude-localhost", true, "whether to exclude localhost endpoints")
```

Reply:
You marked this comment as resolved but It seems that you haven't resolved this comment?